core/storage/usermanagement.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package storage defines storage interfaces.
```

Comment:
Is there anything more insightful that could be added here?  It's a bit self-evident otherwise.

Reply:
Not really.  I was going back and forth between 
1. a `storage` package which had all the storage interfaces with names such as `storage.KeySets`.  The downside is that there's two `storage` packages. One with the interfaces in `core`, and one with the implementations in `impl`.

2. Individual packages for each interface with names such as `keysets.Interface` or `keysets.Storage`.   We also end up with `keysets` packages in both `core` and `impl`.

What do you think?